tests/process.issue.test.ts

```diff
@@ -0,0 +1,41 @@
+import { parseGitHubUrl } from "../src/start";
+import { IssueActivity } from "../src/issue-activity";
+import { Processor } from "../src/parser/processor";
+import { UserExtractorModule } from "../src/parser/user-extractor-module";
+import { server } from "./__mocks__/node";
+import { DataPurgeModule } from "../src/parser/data-purge-module";
+import userCommentResults from "./__mocks__/results/user-comment-results.json";
+import dataPurgeResults from "./__mocks__/results/data-purge-result.json";
+
+const issueUrl = process.env.TEST_ISSUE_URL || "https://github.com/ubiquibot/comment-incentives/issues/22";
+
+beforeAll(() => server.listen());
+afterEach(() => server.resetHandlers());
+afterAll(() => server.close());
+
+describe("Modules tests", () => {
+  const issue = parseGitHubUrl(issueUrl);
+  const activity = new IssueActivity(issue);
+
+  beforeAll(async () => {
+    await activity.init();
+  });
+
+  it("Should extract users from comments", async () => {
+    const logSpy = jest.spyOn(console, "log");
+    const processor = new Processor();
+    processor["_transformers"] = [new UserExtractorModule()];
+    await processor.run(activity);
+    processor.dump();
+    expect(logSpy).toHaveBeenCalledWith(JSON.stringify(userCommentResults, undefined, 2));
```

Comment:
By definition `dump` was just outputting results to stdout, but I could make it return the result as well. Since the console log is what we want to make sure works properly to be reused by other workflows, I thought it would make sense to test it that way because potentially if we forget other logs somewhere that would mess up the output for the next Action step.